ovs/extensions/services/systemd.py

```diff
@@ -41,6 +41,8 @@ def _get_name(name, client, path=None):
         """
         if Systemd._service_exists(name, client, path):
             return name
+        if Systemd._service_exists(name, client, '/lib/systemd/system/'):
```

Comment:
FYI: see this pull request - https://github.com/openvstorage/openvstorage/pull/23